crates/language/src/language_registry.rs

```diff
@@ -180,6 +180,19 @@ impl LanguageRegistry {
     pub fn test(executor: BackgroundExecutor) -> Self {
         let mut this = Self::new(Task::ready(()), executor);
         this.language_server_download_dir = Some(Path::new("/the-download-dir").into());
+
+        this.add(Arc::new(crate::Language::new(
```

Comment:
Can we just add this to the tests that need it rather than having every test `LanguageRegistry` have Markdown registered?